cristin-import/src/main/java/no/unit/nva/cristin/mapper/Event.java

```diff
@@ -0,0 +1,61 @@
+package no.unit.nva.cristin.mapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import nva.commons.core.JacocoGenerated;
+
+@Builder(
+    builderClassName = "EventBuilder",
+    toBuilder = true,
+    builderMethodName = "builder",
+    buildMethodName = "build",
+    setterPrefix = "with"
+)
+@Getter
+@Setter
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@JsonIgnoreProperties({"arstall", "hendelsetype", "antall_deltakere",
+    "antall_internasjonale_deltakere", "antall_nasjonale_deltakere", "landkode",
+    "institusjonsnr_arrangor", "avdnr_arrangor", "undavdnr_arrangor", "gruppenr_arrangor", "stedangivelse",
+    "utbredelsesomrade", "url", "personlopenr_arrangor"})
+public class Event {
```

Comment:
There is already 5 other types of "Event" in this system. Can this be named something along the name presentationEvent?